setup.cfg

```diff
@@ -35,6 +35,7 @@ install_requires =
     pandas
     platformdirs
     prettytable
+    astropy
```

Comment:
I looked at different libraries, it seemed to me that Pint, unyt, and astropy were more robust than the other libraries. At the end I chose astropy because it was the easiest to convert between different unit systems when `new_unit` is not specified. With other libraries, the conversion between different systems seems to require the user to specify 'new_unit`.